ngraph.mk

```diff
@@ -148,12 +167,11 @@ $(NGRAPH_BRIDGE_OBJ): %.o: ngraph $(NGRAPH_BRIDGE_SRC)
   NGRAPH_COMMON_LIBRARY_LDFLAGS_ := \
     -ltbb \
     -liomp5 \
-    -lmklml_intel \
+    -lmklml \
```

Comment:
Note to self: Either (a) make this library choice OS-specific, or (b) make it possible to use `libmkml_intel` on OS X.